### PR TITLE
`<Text/>` - expose tooltip props

### PR DESCRIPTION
**Pull request by mykas**
This PR will expose tooltip props for our Text component usage that wants to control ellipsis floating behaviour. https://github.com/wix/wix-style-react/issues/3664

### CODE REVIEW

**Comment by ipanasenko** [comment]
Is it possible to also add `zIndex` prop? Do you need an issue for this?

File: src/Text/ProxyText.js [hunk 1]
```diff
@@ -1,21 +1,72 @@
...
+  ]),
+  /** `ellipsis` prop. Whether to enable the flip behaviour. This behaviour is used to flip the Tooltips placement when it starts to overlap the target element. */
+  flip: bool,
+  /** `ellipsis` prop. Whether to enable the fixed behaviour. This behaviour is used to keep the Tooltip at it's original placement even when it's being positioned outside the boundary. */
+  fixed: bool,
+  /** `ellipsis` prop. Tooltip content placement in relation to target element */
+  placement: string,
+  /** `ellipsis` prop. Tooltip timeout value. */
+  timeout: number,
+  /** `ellipsis` prop. Tooltip content max width value. */
```
Comment (ipanasenko) [inline src/Text/ProxyText.js]: why does it say "`ellipsis` prop" everywhere?

Comment (mykas) [reply src/Text/ProxyText.js]: This is to distinguish that this effects only the ellipsis. We are currently developing a feature in our storybook to split exposed tooltip props table from the main api table.

**Comment by mykas** [comment]
It would be better if you could open a PR for this feature.

File: src/Text/ProxyText.js (hunk 1 above)
Comment (ipanasenko) [reply src/Text/ProxyText.js]: Oh, okay. Makes sense

**Comment by ipanasenko** [comment]
https://github.com/wix/wix-style-react/pull/3725

**Pull request merged**